src/widgets/CapabilityText.tsx

```diff
@@ -60,6 +60,9 @@ export class CapabilityText {
         [ElementWidgetCapabilities.CanChangeViewedRoom]: {
             [GENERIC_WIDGET_KIND]: _td("Change which room you're viewing"),
         },
+        [MatrixCapabilities.MSC2931Navigate]: {
+            [GENERIC_WIDGET_KIND]: _td("Change which room you're viewing and use permalinks"),
```

Comment:
What does "use permalinks" mean...? Is it something like "jump to specific events"? "Use permalinks" seems to technical / implementation focused for a permissions request UI.

Reply:
The general case is meant to be event permalinks, but the phrasing here is indeed overloading "permalinks" to include rooms, users, etc. Not sure there's a term that can describe it all in a concise way (that also doesn't scare people).

Typically this would be "can instruct your client to look at something else", but that is overly scary and vague. Definitely open to suggestions - I haven't thought of anything much better in the last couple weeks :(

Reply:
Have made an attempt at a new string.

Reply:
New string is much better, thanks! 😄